README.md

```diff
@@ -1,4 +1,8 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Development Environment Setup
+
+First, install the package manager `yarn` from your system package manager or with `npm install -g yarn`. Ensure you have git submodules set up for this project with `git submodule update --init`.
```

Comment:
For the yarn installation instructions, we should just link to this page. The yarn team discourages using `npm` to install yarn, for *reasons*.
https://classic.yarnpkg.com/en/docs/install

Reply:
I'd put the git submodules instruction on  a new line and maybe put the steps in bullet points.
Good job adding that. I always get confused when I freshly clone the repo, and have to look that command up.

Reply:
`git clone --recurse-submodules ` also does the trick

Reply:
Yep! I intentionally used a separate submodule command so it's the same to fix a clone you forgot to do recursively as it is to do it initially, but that works too.

Reply:
Re: the yarn thing, I mistakenly thought we were using Yarn 2, for which that is the recommendation. I changed it.